api/RTCStatsReport.json

```
@@ -4576,6 +4576,111 @@
             }
           }
         },
+        "frames": {
+          "__compat": {
+            "description": "<code>frames</code> in 'media-source' stats",
```

Note, these are video only, and there are a few that are audio-only. Should we mention this somehow in the description? And if so, how?

---

Hmm, maybe something like `<code>frames</code> in 'media-source' stats for video`?

---

I think that's worth doing.

```suggestion
            "description": "<code>frames</code> in 'media-source' stats for video",
```